xml/System.Text.RegularExpressions/GroupCollection.xml

```diff
@@ -1316,10 +1373,11 @@
         <Parameter Name="index" Type="System.Int32" Index="0" FrameworkAlternate="netcore-2.0;netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="index">To be added.</param>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <param name="index">The zero-index position of the group in the group collection.</param>
+        <summary>Gets the group in the desired position.</summary>
+        <value>The group in the desired position.</value>
         <remarks>To be added.</remarks>
```

Comment:
```suggestion
This member is an explicit interface member implementation. It can be used only when the <xref:System.Text.RegularExpressions.GroupCollection> instance is cast to an <xref:System.Collections.IList> interface. ```

Reply:
```suggestion
         <remarks>
           <format type="text/markdown"><![CDATA[  

## Remarks

This member is an explicit interface member implementation. It can be used only when the <xref:System.Text.RegularExpressions.GroupCollection> instance is cast to an <xref:System.Collections.IList> interface. 

           ]]></format>
         </remarks>
```